src/Entity/Tradeability/Shipment.php

```diff
@@ -83,11 +83,6 @@ public function toNode(DOMDocument $document = null)
 
         $node = $document->createElement('Shipment');
 
-        // Objects first
-        if ($this->getAdditionalInsurance() instanceof AdditionalInsurance) {
```

Comment:
Ah, yes. Can you also remove the property + getter/setter? I think it was work in progress that got forgotton, if that class does not exist.